### PR TITLE
한줄소개 수정

### PR DESCRIPTION
**Pull request by raccoonyy**

### CODE REVIEW

**Review by KimWooHyun — APPROVED** [review]
LGTM

File: index.html [hunk 1]
```diff
@@ -396,7 +396,7 @@ <h2 class="title">준비위원회</h2>
                     <figcaption class="flex flex-ai-center flex-wrap">
                       <div class="col-m-12 col-p-12">
                         <p class="p-name">김승호</p>
-                        <p class="p-contents">부위원장</p>
+                        <p class="p-contents"><a href="https://twitter.com/raccoonyy">raccoony</a></p>
```
Comment (KimWooHyun) [inline index.html]: !!!

**Review by KimWooHyun — CHANGES_REQUESTED** [review]

**Review by KimWooHyun — APPROVED** [review]
💯

File: index.html (hunk 1 above)
Comment (raccoonyy) [reply index.html]: fixed! https://github.com/devdjangokr/dev-django-korea-2018/pull/20/commits/d93a8a87ae7e1cf1cbec0a00031eb8a711325db1

**Pull request merged**